assignments/array-methods.js

```diff
@@ -56,28 +56,84 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach((runner) => {
+    fullName.push(`${runner.first_name} ${runner.last_name}`);
+})
+
 console.log(fullName);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map((runner) => {
+    allCaps.push(runner.first_name.toUpperCase());
+})
+
 console.log(allCaps); 
 
+
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+
+runners.filter((runner) => {
+    if (runner.shirt_size === "L") {
+        largeShirts.push(`${runner.first_name} ${runner.last_name}`);
+    }
+})
+
 console.log(largeShirts);
 
+// Alternative solution just for fun
+
+const diffSize = runners.filter((runner) => {
+    if (runner.shirt_size === "L") return runner;
+}).map((runner) => {
+    return `${runner.first_name} ${runner.last_name}`;
+})
+
+console.log(diffSize);
+
```

Comment:
Good use of array methods! :1st_place_medal:

Reply:
Thanks! Before coming to LS I was intimidated by these methods - now chaining makes sense to me. :)